### PR TITLE
Upgrade to Ember 2.16

### PR DESCRIPTION
**Pull request by yawboakye**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
-# Super Rentals
+# super-rentals
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
-
-This is the working repository for the Super Rentals tutorial,
-which you can check out at https://guides.emberjs.com/current/tutorial/ember-cli/.
+This README outlines the details of collaborating on this Ember application.
```
Comment (sivakumar-kailasam) [inline README.md]: Can we restore this block from the original?

Comment (yawboakye) [reply README.md]: Done! @sivakumar-kailasam

**Comment by sivakumar-kailasam** [comment]
Thanks @yawboakye!

**Comment by acorncom** [comment]
👏 🎉

**Pull request merged**